packages/frontend-app-api/src/wiring/createApp.tsx

```diff
@@ -366,13 +369,13 @@ function createApiHolder(
 ): ApiHolder {
   const factoryRegistry = new ApiFactoryRegistry();
 
-  const apiFactories =
+  const pluginApis =
     coreExtension.attachments
       .get('apis')
       ?.map(e => e.getData(coreExtensionData.apiFactory))
       .filter((x): x is AnyApiFactory => !!x) ?? [];
 
-  for (const factory of apiFactories) {
+  for (const factory of [...defaultApis, ...pluginApis]) {
```

Comment:
>[!Note]
The search plugin uses `Discovery API`, so created it to provide a default implementation.